connectors/cassandra/datastore_common_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cassandra_test
+
+import (
+	"context"
+
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/uber-go/dosa"
+	. "github.com/uber-go/dosa/connectors/cassandra"
+)
+
+const (
+	keyspace       = "datastore_test"
+	tableName      = "testentity"
+	uuidKeyField   = "uuidkeyfield"
+	uuidField      = "uuidfield"
+	stringKeyField = "stringkeyfield"
+	stringField    = "stringfield"
+	int32Field     = "int32field"
+	int64KeyField  = "int64keyfield"
+	int64Field     = "int64field"
+	doubleField    = "doublefield"
+	blobField      = "blobfield"
+	timestampField = "timestampfield"
+	boolField      = "boolfield"
+)
+
+var testStore *Connector
+
+func initTestStore(t *testing.T) error {
```

Comment:
I don't think you need to return an error here, since it doesn't seem possible for this function to return an error?